### PR TITLE
Feature/pl20 3200/restructure java helpers code

### PR DESCRIPTION
**Pull request by jenish4096**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,27 +25,23 @@ dependencies {
...
 - Know your tenant (BIDTenant) `dns` and `communityName`
-- Initialize SDK
-```
-BIDSDK.getInstance().setupTenant(<your tenant>, "<your license key>");
-```
 
 - Request OTP
 ```
-BIDOtpResponse otpResponse = BIDOTP.requestOTP("<username>", "<emailTo>", "<smsTo>", "1");
+BIDOtpResponse otpResponse = BIDOTP.requestOTP({ "dns": "<dns>", "communityName": "<communityName>", "licenseKey": "<licenseKey>" },"<username>", "<emailTo>", "<smsTo>", "1");
```
Comment (manik-1kosmos) [inline README.md]: @jenish4096 this appears to be JS code structure.. can we please fix this.

File: README.md [hunk 2]
```diff
@@ -25,27 +25,23 @@ dependencies {
...
 ```
 
 - Verify OTP
 ```
-BIDOtpVerifyResult result = BIDOTP.verifyOTP("<username>", "<otpcode>");
+BIDOtpVerifyResult result = BIDOTP.verifyOTP({ "dns": "<dns>", "communityName": "<communityName>", "licenseKey": "<licenseKey>" }, "<username>", "<otpcode>");
```
Comment (manik-1kosmos) [inline README.md]: same here

File: README.md [hunk 3]
```diff
@@ -25,27 +25,23 @@ dependencies {
...
 ```
 
 - Create new UWL2.0 session
 ```
-BIDSession session = BIDSessions.createNewSession(null, null);
+BIDSession session = BIDSessions.createNewSession({ "dns": "<dns>", "communityName": "<communityName>", "licenseKey": "<licenseKey>" }, null, null);
```
Comment (manik-1kosmos) [inline README.md]: same here

File: README.md [hunk 4]
```diff
@@ -25,27 +25,23 @@ dependencies {
...
 ```
 
 - Poll for UWL2.0 session response
 ```
-BIDSessionResponse response = BIDSessions.pollSession("<sessionId>", true, true);
+BIDSessionResponse response = BIDSessions.pollSession({ "dns": "<dns>", "communityName": "<communityName>", "licenseKey": "<licenseKey>" }, "<sessionId>", true, true);
```
Comment (manik-1kosmos) [inline README.md]: same here

File: README.md (hunk 1 above)
Comment (jenish4096) [reply README.md:31]: @manik-1kosmos can we do like below?

```
Map<String, Object> tenantInfo = new HashMap<>();
tenantInfo.put("dns", <dns>);
tenantInfo.put("communityName", <communityName>);
tenantInfo.put("licenseKey", <licenseKey>);

BIDOtpResponse otpResponse = BIDOTP.requestOTP(tenantInfo, "<username>", "<emailTo>", "<smsTo>", "1");
```

File: README.md [hunk 5]
```diff
@@ -25,27 +25,48 @@ dependencies {
...
 - Know your tenant (BIDTenant) `dns` and `communityName`
-- Initialize SDK
-```
-BIDSDK.getInstance().setupTenant(<your tenant>, "<your license key>");
-```
 
 - Request OTP
 ```
-BIDOtpResponse otpResponse = BIDOTP.requestOTP("<username>", "<emailTo>", "<smsTo>", "1");
+BIDTenantInfo tenantInfo = new BIDTenantInfo();
```
Comment (manik-1kosmos) [inline README.md]: @jenish4096 add a constructor to BIDTenantInfo to initialize the object in single line please.

Comment (jenish4096) [reply README.md]: okay

**Pull request merged**